src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -136,7 +136,9 @@ function MoneyRequestView({report, betas, parentReport, policyCategories, should
         if (!isDistanceRequest) {
             amountDescription += ` • ${translate('iou.cash')}`;
         }
-        if (isSettled) {
+        if (action && action.originalMessage.cancellationReason === CONST.IOU.CANCEL_REASON.PAYMENT_EXPIRED) {
```

Comment:
```suggestion
        if (action && action.originalMessage && action.originalMessage.cancellationReason === CONST.IOU.CANCEL_REASON.PAYMENT_EXPIRED) {
```

<img width="1713" alt="Screenshot 2023-11-13 at 12 27 45 PM" src="https://github.com/Expensify/App/assets/12268372/84d5207b-63ea-44af-b945-3eb27f1e8cf1">